flake.nix

```
@@ -701,7 +711,7 @@
 
             outputs = [ "out" "dev" "doc" ];
 
-            nativeBuildInputs = nativeBuildDeps;
+            nativeBuildInputs = nativeBuildDeps ++ [ clang-tools ];
```

```suggestion
            nativeBuildInputs = nativeBuildDeps;
```

I gues this sneaked in accidentally